core/services/ocr2/plugins/ocr2keeper/evm21/logprovider/integration_test.go

```diff
@@ -175,7 +175,7 @@ func TestIntegration_LogEventProvider_UpdateConfig(t *testing.T) {
 			TriggerConfig: cfg,
 			UpdateBlock:   bn.Uint64() - 1,
 		})
-		require.Error(t, err)
+		require.NoError(t, err)
```

Comment:
this seems like the opposite assert now. not sure why this needs to change with the naming changes?

Reply:
We will log (debug) rather than returning an error, therefore the error check is irrelevant